include/vara/Feature/Location.h

```diff
@@ -0,0 +1,46 @@
+#ifndef VARA_FEATURE_LOCATION_H
+#define VARA_FEATURE_LOCATION_H
+
+#include <optional>
+#include <sstream>
+#include <string>
+
+namespace vara::feature {
+
+class Location {
+private:
+  std::string Path;
+  std::optional<std::pair<int, int>> Start;
```

Comment:
```suggestion
  std::optional<HelperClassFoo> Start;
```

```
class HelperClassFoo : private std::pair<int, int> {
  int getLineNumber() { ... }
  ...
}
```

We should add a nested class here to have a type with a name and accessor methods that have meaning full names, otherwise, the code is quite hard to understand.